Iteam1337/brevduvor
# feat: persist user preferences

**Issue:** Closes #41
**Intent:**
Adds language preference to User

We decided against adding a separate table to hold user preferences in our DB after internal discussions.
**How to test (optional):**
Register a user
Add the received token in header `{authorization: "Bearer ${token}"}`
Use `updateUserLanguage` mutation
Check the users table to see that language field changes
